### DISCUSSION TITLE
Error importing a txt file

### DISCUSSION THREAD

**Discussion by andrefahl** [Julian-Hochhaus/LG4X-V2#96] [General]
When I try to import the txt file with data of my measure I got the following error : The input file is not in the correct format!. The following traceback may help to solve the issue:\n *******************\nTraceback (most recent call last): File "e:\LG4X\LG4X-V2-master\Python\main.py", line 2029, in plot self.df = np.loadtxt(str(self.comboBox_file.currentText()), delimiter='\t', skiprows=1) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "C:\Users\fahl_\AppData\Roaming\Python\Python312\site-packages\numpy\lib\npyio.py", line 1373, in loadtxt arr = _read(fname, dtype=dtype, comment=comment, delimiter=delimiter, ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "C:\Users\fahl_\AppData\Roaming\Python\Python312\site-packages\numpy\lib\npyio.py", line 1016, in _read arr = _load_from_filelike( ^^^^^^^^^^^^^^^^^^^^ ValueError: could not convert string '8.924015324999998484e+03 2.540000000000000000e+02 1.776712260248795774e-03' to float64 at row 0, column 1. Somebody please, could help me?

**Comment by Julian-Hochhaus** [comment]
For the solution, please refer to Issue #97. This link is provided to guide others to the solution.
